test/index.js

```diff
@@ -8,6 +8,18 @@ import {
   GatewayProvider
 } from '../src/index.js';
 
+// React v15 uses commment nodes for empty components instead of `<noscript />`
+// so in those cases we can skip rendering children
+const emptyComponent = (function emptyComponent() {
+  const [major, minor, patch] = React.version.split('.').map(Number);
+
+  if (major >= 15) {
+    return null;
+  }
+
+  return <noscript />;
+})();
```

Comment:
So it turns out that rendering `null` from a stateless component is only allowed in React 15 ([issue](https://github.com/facebook/react/issues/5355)).  I can do something like this:

```javascript
class Empty extends React.Component {
 render() {
   return null;
 }
}
```

But because it's a pure component class, it's getting transformed by the `babel-plugin-react-pure-components` back to a stateless function before the tests run.  We could just force it with something like this:

```javascript
class Empty extends React.Component {
  notPure() {}
  render() {
    return null;
  }
}
```

Not sure I love that, but it does work.  What do you think?